src/commands/deploy.js

```diff
@@ -90,22 +92,21 @@ const createEmbulkConfig = (mashrConfigObj) => {
   return yaml.safeDump(embulkConfig);
 }
 
-const createCrontab = (runCommand) => {
+const createEmbulkScript = (runCommand) => {
   // TODO: place logs in stackdriver
   // TODO: what to do with logs? Does the log file get too large?
   // diff file run from root of container. Can't use it after?
-  let crontabCommand = runCommand;
-  crontabCommand = crontabCommand.replace(
-  'embulk_config.yml', '/app/mashr/embulk_config.yml.liquid');
-
-  const crontab =
-`PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
-DATE=date +"%Y-%m-%d-%H_%M_%S_%3N;
-export DATE
-* * * * * ${crontabCommand} >> /var/log/cron.log 2>&1
-# An empty line is required at the end of this file for a valid cron file.
+  runCommand = runCommand.replace(
+    'embulk_config.yml', '/root/mashr/embulk_config.yml.liquid');
+
+  const script =
+`#!/bin/bash
+export DATE=$(date +"%Y-%m-%dT%H-%M-%S-%3N")
```

Comment:
Where in the files does the `DATE` variable get used? I believe it's appended to the name of the file, but I'm not sure where. Could this be done inside this script to make it more obvious to someone reading the code?

Reply:
This is set in the `path_prefix` field in the final `embulk_config` template file. It's going to be used for the name of the data file that's produced and stored in Cloud Storage.